archinstall/lib/output.py

```diff
@@ -137,26 +137,28 @@ def log(message: str, level: int = logging.DEBUG) -> None:
 
 def check_log_permissions():
 	filename = storage.get('LOG_FILE', None)
+	log_dir = storage.get('LOG_PATH', Path('./'))
 
 	if not filename:
-		return
+		raise ValueError('No log file name defined')
 
-	log_dir = storage.get('LOG_PATH', Path('./'))
-	absolute_logfile = log_dir / filename
+	log_file = log_dir / filename
 
 	try:
 		log_dir.mkdir(exist_ok=True, parents=True)
-		with absolute_logfile.open('a') as fp:
+		log_file.touch(exist_ok=True)
+
+		with log_file.open('a') as fp:
 			fp.write('')
```

Comment:
```suggestion

```

Is this still needed now that `log_file.touch(exist_ok=True)` has been added just above this?